dagshub/upload/wrapper.py

```diff
@@ -194,10 +195,10 @@ def upload(
 
         """
         if os.path.isdir(local_path):
-            dir_to_upload = self.directory(remote_path)
+            dir_to_upload = self.directory(remote_path if remote_path is not None else DEFAULT_REMOTE_PATH)
             dir_to_upload.add_dir(local_path, commit_message=commit_message, **kwargs)
```

Comment:
This is perfectly fine and thank you for actually doing this :)

I think for this to work perfectly it'd be cool to get the directory name, and upload to `/data/{name_of_the_directory}`.

BTW, I'm going to work a little bit on the Repo class in general, so I might implement the behavior that the upload happens relative to the repo root (if the script is called in a git repository).
So, for example, if you have your repo in `~/repos/cool-repo/`, do a script with cwd set somewhere inside of the repo AND then you upload a folder `~/repos/cool-repo/some/other/folder/with/data`, we'll upload the folder to `some/other/folder/with/data`.
BUT, if cwd is not inside of a repository, then upload it to the `/data/src/` folder just like you implemented. Let me know what you think about it, I'd love to hear your thoughts :)

Reply:
Actually thats a cool idea! @kbolashev